Solves the compilation issue with older cairo (tested with EL-5's 1.2.8)

Lubomir Rintel <lkundrak@v3.sk>

Index: src/extension/internal/cairo-ps-out.cpp
===================================================================
--- src/extension/internal/cairo-ps-out.cpp	(revision 20786)
+++ src/extension/internal/cairo-ps-out.cpp	(working copy)
@@ -38,6 +38,11 @@
 
 #include "io/sys.h"
 
+#if CAIRO_VERSION < CAIRO_VERSION_ENCODE(1, 5, 1)
+#  define CAIRO_PS_LEVEL_2 0
+#  define CAIRO_PS_LEVEL_3 1
+#endif
+
 namespace Inkscape {
 namespace Extension {
 namespace Internal {
